.mailmap

```diff
@@ -1519,6 +1519,7 @@ erdOne <36414270+erdOne@users.noreply.github.com>
 extraymond <extraymond@gmail.com>
 faizan2700 <syedfaizan824@gmail.com>
 foice <foice.news@gmail.com>
+gnrraphi <raphael.lehner@gmail.com>
```

Comment:
Is this the name that you want to have recorded in the AUTHORS file?

You could add this as
```
Raphael Lehner <raphael.lehner@gmail.com> gnrraphi <raphael.lehner@gmail.com>
```
Then the .mailmap file will show Raphael Lehner but the mailmap check scrpt will understand that commits recorded as `gnrraphi <raphael.lehner@gmail.com>` belong to that same person.